nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/CheckInDocument.java

```diff
@@ -54,7 +59,7 @@
     protected String versionVarName;
 
     protected VersioningOption getVersioningOption() {
-        return "major".equalsIgnoreCase(version) ? VersioningOption.MAJOR : VersioningOption.MINOR;
+        return options.getOrDefault(version.toLowerCase(), VersioningOption.MINOR);
```

Comment:
Please don't use `toLowerCase()` but `toLowerCase(Locale.ENGLISH)`. See the Javadoc of `toLowerCase()` for why.